Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.10.16-alpine3.21
+LABEL maintainer="gnonasis@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /backend
+
+COPY requirements.txt requirements.txt
+
+RUN apk add --no-cache gcc musl-dev libffi-dev \
+    && pip install --no-cache-dir -r requirements.txt\
+    && apk del gcc musl-dev libffi-dev
+
+COPY . .

```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if there are unnecessary files in the context. Ensure your `.dockerignore` file is properly configured to exclude files that are not needed in the image.